### PR TITLE
6248 Fixed the geoloc zoom slider snap bug

### PR DESCRIPTION
**Pull request by ethteck**

### CODE REVIEW

File: Core/src/org/sleuthkit/autopsy/geolocation/MapPanel.form [hunk 1]
```diff
@@ -56,10 +56,10 @@
                 <Property name="orientation" type="int" value="1"/>
                 <Property name="paintTicks" type="boolean" value="true"/>
                 <Property name="snapToTicks" type="boolean" value="true"/>
+                <Property name="inverted" type="boolean" value="true"/>
                 <Property name="minimumSize" type="java.awt.Dimension" editor="org.netbeans.beaninfo.editors.DimensionEditor">
                   <Dimension value="[35, 100]"/>
                 </Property>
-                <Property name="opaque" type="boolean" value="false"/>
```
Comment (kellykelly3) [inline Core/src/org/sleuthkit/autopsy/geolocation/MapPanel.form:62]: Why was this changed? It looks much better as opaque.  That was an intentional design decision.  What was the thought behind changing it?

Comment (ethteck) [reply Core/src/org/sleuthkit/autopsy/geolocation/MapPanel.form:62]: It wasn't consciously changed. This happened automatically by the editor. I'm looking into changing it currently.

**Comment by ethteck** [comment]
@kellykelly3 I reverted the form code changes. The PR should be more straightforward now.
